UnitTests/BraintreePayPalTests/BTPayPalClient_Tests.swift

```diff
@@ -1028,4 +1028,34 @@ class BTPayPalClient_Tests: XCTestCase {
 
         XCTAssertFalse(mockAPIClient.postedIsVaultRequest)
     }
+
+    func test_payPalAppNotInstalled() {
+        let fakeApplication = FakeApplication()
+        payPalClient.application = fakeApplication
+        fakeApplication.cannedCanOpenURL = false
+
+        XCTAssertEqual(fakeApplication.isPayPalAppInstalled(), payPalClient.application.isPayPalAppInstalled())
```

Comment:
Could we instead be a bit more explicit in these with something like:
```swift
XCTAssertFalse(payPalClient.application.isPayPalAppInstalled())
```

Reply:
Thanks for the suggestions!